content/hardware/05.pro-solutions/solutions-and-kits/opta/tutorials/getting-started/content.md

```diff
@@ -186,8 +186,8 @@ The coils of each relay correspond to pins D0 to D3 as follows:
 | :--------: | :---: | :----: |
 | OUTPUT 1   | D0    | RELAY1 |
 | OUTPUT 2   | D1    | RELAY2 |
-| OUTPUT 3   | D3    | RELAY3 |
-| OUTPUT 4   | D4    | RELAY4 |
+| OUTPUT 3   | D2    | RELAY3 |
+| OUTPUT 4   | D3    | RELAY4 |
```

Comment:
Is this correct?

check: https://github.com/arduino/ArduinoCore-mbed/blob/master/variants/OPTA/pins_arduino.h#L36-L39
![image](https://user-images.githubusercontent.com/11246294/220618041-b3c3e051-3228-41fb-b8b9-3f5f6d159c1a.png)

```
#define LED_D0      (7u) - D7
#define LED_D1      (9u) - D9
#define LED_D2      (8u) - PIN_SPI_MOSI  
#define LED_D3      (153u)
```

Reply:
Hello @marqdevx 

The change is not related with those pins, it is related with:

![image](https://user-images.githubusercontent.com/7687624/220633382-d4bdc878-f7ca-4261-9d4d-6d87009cb919.png)

`#define RELAY1 (D0)
#define RELAY2 (D1)
#define RELAY3 (D2)
#define RELAY4 (D3)`

Which as you can double check was wrong, and that is the reason of the urgent change 

Thanks :)

Reply:
You are right! my fault, approving